psse/psse-model/src/main/java/com/powsybl/psse/model/pf/PsseTransformerWinding.java

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.psse.model.pf;
+
+import com.powsybl.psse.model.PsseVersioned;
+import com.powsybl.psse.model.Revision;
+import com.univocity.parsers.annotations.Parsed;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public class PsseTransformerWinding extends PsseVersioned {
+    @Override
+    public void setModel(PssePowerFlowModel model) {
+        super.setModel(model);
+    }
```

Comment:
To be removed

Reply:
Done. Override was needed when trying to use `Rates` nested in each `TransformerWinding` instead of a `Transformer` level. Now it is not required.